README.md

```diff
@@ -1,6 +1,24 @@
-# data-update-github
+# git-ops-api
 
-A remote data change triggers the issue of a pull request to a specified Github repository.
+This repository supports GitHub operation API for manipulating branches and files in remote GitHub repositories.
+This API makes use of [GitHub REST API](https://docs.github.com/en/rest) and [GitHub GraphQL API](https://docs.github.com/en/graphql).
+
+## Git Operation API
+
+* Script: lib/gitOpsApi.js
+
+Includes API for operating remote Github branches and files. All API functions return a promise. Refer to the JSDoc
+in the script for the detail of parameters.
+
+* getBranchRef(octokit, options)
+* getAllBranches(octokit, options)
+* createBranch(octokit, options)
```

Comment:
Thanks, this is looking really great! The new getFileLastCommit method should be listed here

Reply:
Good catch. Addressed.